scripts/Util.py

```diff
@@ -3239,11 +3239,11 @@ def getUWPUserModelId(self):
         return "ice-uwp-controller.cpp_3qjctahehqazm"
 
     def getUWPPackageFullName(self, platform):
-        return "{0}_1.0.0.0_{1}__3qjctahehqazm".format(self.getUWPPackageName(),
+        return "{0}_1.1.0.0_{1}__3qjctahehqazm".format(self.getUWPPackageName(),
```

Comment:
Obviously we should not support UWP.

Reply:
Yes, this file is a copy of `ice/Util.py` file and contains support for many platforms that datastorm doesn't support (iOS, UWP, Android, etc). I don't think we should have a datastorm specific version, too much trouble to maintain.